src/parser/mod.rs

```diff
@@ -73,7 +75,7 @@ fn statement<'a, 'b>(input: Input<'a, 'b>) -> AstResult<'a, 'b> {
     match input[0].0 {
         Token::ParenthesisLeft | Token::Identifier(_) => or(&[definition, assignment, expression], "statement")(input),
         Token::Type => or(&[type_definition, type_alias], "statement")(input),
-        Token::Import => import(input),
+        Token::Import => or(&[import_defs], "import")(input),
```

Comment:
`or(&[import_defs], ..)(input)` should be equivalent to just `import_defs(input)` with a slightly different error message.